src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/Edit.cshtml

```diff
@@ -156,8 +156,8 @@
         <button class="btn btn-danger delete" type="submit" name="submit.Delete" value="Delete" data-url-af="RemoveUrl">@T["Delete"]</button>
     </div>
 </form>
-
-<script at="Foot">
+<script asp-name="Sortable"></script>
+<script at="Foot" depends-on="Sortable">
```

Comment:
Without this, the UI is still sortable, despite not having `Sortable` included. Why does it work? Do we actually need `Sortable`?

Reply:
Without this, it does not work. Unless other module include it. 

But if you create a site using SaaS recipe, then enable content-types feature and try to sort the content part. That would fail. Including it ensure the script runs with no problems

Reply:
@Piedone 
Another problem with sortable is that it was being complied in TheAdmin.js and one can still enable it using dependencies like we do in the list-part and other places. So I removed Sortable from being complied in TheAdmin.js. But added Sortable dependency to TheAdmin.js so anyone that is using TheAdmin.js will continue to have Sortable, but at least we won't have the same script injected into multiple places.

Also I removed `<script asp-name="Sortable"></script>` since we don't do the same this in other places.

Everything seems to be working as expected now.

Reply:
Ah yeah, that's why it was working for me, without the script being included separately. Great!